Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -24,31 +24,133 @@
 #if ENABLE_SWIFTUI
 
 import Foundation
+internal import Observation
```

Comment:
I think this actually would need to be `public` too, since `WebPage_v0` conforms to `Observable` (via macro expansion). It’s probably moot so long as you also `public import SwiftUI` since SwiftUI itself re-exports Observation though.